static/js/protein/chart.ts

```diff
@@ -905,6 +907,10 @@ export function drawDiseaseGeneAssocChart(
     .attr("width", (d) => x(d.value))
     .attr("height", y.bandwidth())
     .style("fill", BAR_COLOR)
+    .on("click", function (d) {
+      console.log(d.id);
+      window.location.href = `${GRAPH_EXPLORER_REDIRECT}${d.id}`;
```

Comment:
we usually want to use window.open(<url>) which opens the url in a new tab or window.open(<url>, "_self") to open in current tab